src/main/resources/META-INF/plugin.xml

```diff
@@ -1,10 +1,10 @@
 <idea-plugin>
-  <id>org.jboss.tools.intellij.analytics</id>
-  <name>Dependency Analytics</name>
-  <version>1.0</version>
-  <vendor email="developers@redhat.com" url="https://www.redhat.com">Red-Hat</vendor>
+    <id>org.jboss.tools.intellij.analytics</id>
+    <name>Dependency Analytics</name>
+    <version>1.0</version>
```

Comment:
should be version 0.7

Reply:
Corrected it in this PR: https://github.com/redhat-developer/intellij-dependency-analytics/pull/129/files#diff-13198d14dd3abf13abd9cb906331ed7d9663fc081757b6845fd5f3c1850b5693